sycl/include/CL/sycl/detail/device_info.hpp

```diff
@@ -265,6 +265,24 @@ struct get_device_info_cl<T, info::device::parent_device> {
   }
 };
 
+// Specialization for supported subgroup sizes
+template <>
+struct get_device_info_cl<vector_class<size_t>,
+                          info::device::sub_group_sizes> {
+  static vector_class<size_t> _(cl_device_id dev) {
+    size_t resultSize;
```

Comment:
Could you add default initializer?
`size_t resultSize = 0`

Reply:
done